airflow/sensors/external_task_sensor.py

```diff
@@ -21,14 +21,31 @@
 
 from sqlalchemy import func
 
+from airflow.configuration import conf
 from airflow.exceptions import AirflowException
-from airflow.models import DagBag, DagModel, DagRun, TaskInstance
+from airflow.models import BaseOperatorLink, DagBag, DagModel, DagRun, TaskInstance
 from airflow.sensors.base_sensor_operator import BaseSensorOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.utils.session import provide_session
 from airflow.utils.state import State
 
 
+class ExternalTaskLink(BaseOperatorLink):
+    name = 'External DAG'
+
+    def get_link(self, operator, dttm):
+        if isinstance(operator, ExternalTaskSensor):
+            possible_execution_dates = operator.get_possible_target_execution_dates(execution_date=dttm)
```

Comment:
This won't work because of the use of this function.

Check the Qubole Operator and how the test is implemented for it:

https://github.com/apache/airflow/blob/803a87f2b2b564b3e8784de305f5bb4e9578e578/tests/contrib/operators/test_qubole_operator.py#L146-L176

Reply:
The SerializedOperator won't have this function hence it would fail